auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py

```diff
@@ -29,6 +29,20 @@ def get_gpu_available_memory():
 
 
 class Segresnet2dAlgo(BundleAlgo):
+    def pre_check_skip_algo(self):
+        if self.data_stats_files is None:
+            return
+        data_stats = ConfigParser(globals=False)
+        if os.path.exists(str(self.data_stats_files)):
+            data_stats.read_config(str(self.data_stats_files))
+        else:
+            data_stats.update(self.data_stats_files)
+        spacing = data_stats["stats_summary#image_stats#spacing#median"]
+        if len(spacing) > 2:
+            if spacing[-1] < 10 * (spacing[0] + spacing[1]) / 2:
+                self.skip_bundlegen = True
+                self.skip_info = f'2D network is skipped due to median spacing of {spacing}'
```

Comment:
Would be good to have a period `.` in the end of the sentence. Thanks!

Reply:
thanks.